extensions/chghost-3.2.md

```diff
@@ -5,42 +5,68 @@ copyrights:
   -
     name: "Christine Dodrill"
     period: "2013"
-    email: "xena@yolo-swag.com"
+    email: "me@christine.website"
+  -
+    name: "Ryan"
+    period: "2016"
+    email: "ryan@hashbang.sh"
 ---
-The chghost client capability allows a server to directly inform clients about a
-host or user change without having to send a fake quit and join. This capability
-MUST be referred to as `chghost` at capability negotiation time.
 
-When enabled, clients will get the CHGHOST message to designate the host of a
-user changing for clients on common channels with them.
+## Introduction
+
+The `chghost` client capability allows servers to directly inform clients about
+clients changing their host and/or user without having to simulate the client
```

Comment:
Not a fan of and/or. Just "or" is fine. "either their host or user" works too.

Reply:
I think the examples have a case where both are changed so I'm fine with this. I don't think there's any real implementations where this can happen but it might be good just in case.

Reply:
Replaced all instances.